nautobot/circuits/templates/circuits/circuit.html

```diff
@@ -1,2 +1,2 @@
 {% extends 'circuits/circuit_retrieve.html' %}
-{% comment %}2.0 TODO: remove this template, which only exists for backward compatibility with 1.3 and earlier{% endcomment %}
+{% comment %}3.0 TODO: remove this template, which only exists for backward compatibility with 2.4 and earlier{% endcomment %}
```

Comment:
I don't think there's a need to update the comment in the `circuits` templates - the current comment is correct that the circuit templates were changed in 1.4 and the old files really should have been removed in 2.0.